test/Driver/driver_test.jl

```diff
@@ -115,12 +115,12 @@ function main()
     ugeo_abs = FT(7)
     vgeo_abs = FT(5.5)
     Δt = solver_config.dt
-    caₕ = ugeo_abs * (Δt / Δh) + vgeo_abs * (Δt / Δh)
+    ca_h = ugeo_abs * (Δt / Δh) + vgeo_abs * (Δt / Δh)
```

Comment:
What's up with changes to driver_test.jl?

Reply:
For me and some others the subscript did not display properly. What I was seeing before was a ca_box_with_a_questionmark_inside